### PR TITLE
Backlog 38663

### PR DESCRIPTION
**Pull request by lsaikrishna89**

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 2m 27s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: assemblies/core/lib/pom.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
         <jt400.version>6.1</jt400.version>
         <LucidDbClient-minimal.version>0.9.4</LucidDbClient-minimal.version>
         <sapdbc.version>7.4.4</sapdbc.version>
-        <sqlite-jdbc.version>3.27.2.1</sqlite-jdbc.version>
+        <!--<sqlite-jdbc.version>3.27.2.1</sqlite-jdbc.version>-->
```
Comment (smmribeiro) [inline assemblies/core/lib/pom.xml:39]: Why comment and not simply remove this?

**Review by smmribeiro — CHANGES_REQUESTED** [review]
Code does not seem right...

**Comment by buildguy** [comment]
### :warning: Build finished in 50m 22s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 6 test(s) failed:

<details><summary><b>org.pentaho.di.core.compress.snappy.SnappyCompressionInputStreamTest.testClose</b> (click to expand)</summary><p>

```
[EMPTY_INPUT] Cannot decompress empty stream
```

</p></details>

<details><summary><b>org.pentaho.di.core.compress.snappy.SnappyCompressionInputStreamTest.testCtor</b> (click to expand)</summary><p>

```
[EMPTY_INPUT] Cannot decompress empty stream
```

</p></details>

<details><summary><b>org.pentaho.di.core.compress.snappy.SnappyCompressionInputStreamTest.testRead</b> (click to expand)</summary><p>

```
[EMPTY_INPUT] Cannot decompress empty stream
```

</p></details>

<details><summary><b>org.pentaho.di.core.compress.snappy.SnappyCompressionInputStreamTest.getCompressionProvider</b> (click to expand)</summary><p>

```
[EMPTY_INPUT] Cannot decompress empty stream
```

</p></details>

<details><summary><b>org.pentaho.di.core.compress.snappy.SnappyCompressionInputStreamTest.testNextEntry</b> (click to expand)</summary><p>

```
[EMPTY_INPUT] Cannot decompress empty stream
```

</p></details>

<details><summary><b>org.pentaho.di.core.compress.snappy.SnappyCompressionProviderTest.testCreateInputStream</b> (click to expand)</summary><p>

```
[EMPTY_INPUT] Cannot decompress empty stream
```

</p></details>

  
Tests run: **5327**, Failures: **6**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9070/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by smmribeiro** [comment]
This PR is outdated.
Closing.

**Pull request closed without merging**